mteb/tasks/Classification/vie/AmazonCounterfactualVNClassification.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+
+
+class AmazonCounterfactualVNClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="AmazonCounterfactualVNClassification",
+        dataset={
+            "path": "GreenNode/amazon-counterfactual-vn",
+            "revision": "b48bc27d383cfca5b6a47135a52390fa5f66b253"
+        },
+        description=(
+            "A collection of Amazon customer reviews annotated for counterfactual detection pair classification."
+        ),
+        reference="https://arxiv.org/abs/2104.06893",
+        category="s2s",
+        type="Classification",
+        eval_splits=["test"],
+        eval_langs=["vie-Latn"],
+        main_score="accuracy",
+        date=("2025-07-29", "2025-07-30"),
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license="cc-by-sa-4.0",
+        annotations_creators="derived",
+        dialect=[],
+        sample_creation="machine-translated",
```

Comment:
I would make this "machine-translated and LM verified," given the pipeline. I would also describe the verification process in the description.